src/Game.js

```diff
@@ -4,15 +4,18 @@ import data from './data.js';
 import Puzzle from './Puzzle.js';
 
 class Game {
-  constructor() {
+  constructor(player) {
     this.allPuzzles = [];
     this.players = [];
+    this.currentPlayer = player;
+    this.currentRound = 0;
     this.currentPuzzle = null;
   }
 
   beginGame() {
     this.getRandomPuzzle();
     let puzzle1 = new Puzzle(this.currentPuzzle);
+    this.currentPuzzle = puzzle1;
```

Comment:
I'm glad to see this code fixed a lot of your issues referencing puzzle methods!
![bitmoji](https://render.bitstrips.com/v2/cpanel/7652499a-75e7-4a53-b4d2-47726289c68a-a9e3af97-4465-45bc-94f2-3cedf4f9f291-v1.png?transparent=1&palette=1&width=246)